modules/azure_subscription/outputs.tf

```
@@ -0,0 +1,19 @@
+output "service_principal_id" {
+  value       = azuread_service_principal.illumio_sp.object_id
+  description = "The ID of the service principal created for Illumio CloudSecure."
+}
```

This is meant to be used as input for `azure_flow_logs_storage_accounts`'s `service_principal_client_id` variable. Use the same name to make that obvious.
```suggestion
output "service_principal_client_id" {
  value       = azuread_service_principal.illumio_sp.object_id
  description = "The ID of the service principal created for Illumio CloudSecure."
}
```